pages/web/domains/dns_dnssec/guide.it-it.md

```diff
@@ -1,66 +1,87 @@
----
-title: 'Rendere sicuro un dominio con DNSSEC'
-excerpt: 'Come proteggere un dominio dal Cache Poisoning attivando DNSSEC'
-legacy_guide_number: g609
-updated: 2023-04-26
----
+--- 
```

Comment:
Looks like something is wrong here. Not the same coloration from GitHub compared to other versions.
Same thing for pt-pt.
Please double-check

Reply:
all good on vscode